book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_reader :rentals
```

Comment:
- The rental variable can also be modified hence we should equally have Getters and Setters for @rentals